### ISSUE TITLE
The Kidnapping of Boo *possible* Bug

### ISSUE THREAD

**Issue by Frenzgyn** [Pocket-Plane-Group/UnfinishedBusiness#40]
I noticed a strange bug after I resolve the quest (Fighting Elyanna guards or Peacefully resolved).

Minsc get back Boo but keeps Boo's scraps (WWEBOO.ITM) and the dialogue when you kick him from the party bugs out: once removed from the party he comes to the player to initiate dialogue, but nothing happens, as if the script cannot decide which dialogue should trigger. 
I can replicate if doing the fight again and fix it removing WWEBOO.IT with NI, unfortunately it's a semi-old EET install ( with updated version from end of 2022 mostly), so any kind of strange interaction could happen but I think it is worth mentioning that I use "move Boo's to Minsc's inventory" tweak from Tweak Anthology, maye there is some interaction between the two. 
When you resolve it peacefully, you got a second Boo in Minsc's inventory too, but it doesn't seem to break anything except for wasting a slot, removing Boo's scraps still fix the dialogue.

I didn't notice the issue mentioned (or fixed) in any changelog since then, so hopefully it still applies. 
I've not yet tested it on a fresh install or just on BG2EE.

**Comment by AngelGryph** [comment]
Confirmed, there are compatibility issues between Minsc's quest in UB and the Tweaks (formerly SCS) component that moves Boo to Minsc's inventory, something which the UB scripts don't anticipate.  I'll probably have to commune with CamDawg and/or DavidW on how we can make this as robust as possible, but I'll make the suggested dialog changes.

**Comment by Frenzgyn** [comment]
Never tried the quest without CDTweaks (I don't use SCS at the moment), does Boo's scraps too go in the quickslot by default wit UB? 

Anyway, thank you for the time you spend supporting all the older mods.

**Comment by AngelGryph** [comment]
Yes, oBG2 did not have the TransformItem action, so the change was implemented through a spell using opcode 143 (Create Item in Slot) instead.
